.github/workflows/contract-integration.yaml

```diff
@@ -20,16 +20,42 @@ jobs:
           python-version: 3
       - run: |
           pip install -e .
-          cd provider/*project
+          cd lookoutvision/*project
           curl -s -L https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip -o sam.zip
           unzip -q sam.zip -d sam-install
           sudo ./sam-install/install
           sam local start-lambda &
-          pip install --quiet cloudformation-cli-java-plugin
+          pip install --quiet cloudformation-cli-java-plugin cloudformation-cli-python-plugin
           mvn package --no-transfer-progress -Dmaven.test.skip=true
           cfn test --enforce-timeout 240
           ! grep -q -E -i '(accessKeyId|secretAccessKey|sessionToken)' rpdk.log*
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_REGION: us-east-1
+  contract-integration-python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: aws-cloudformation/aws-cloudformation-resource-providers-frauddetector
+          path: frauddetector
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3
+      - run: |
+          pip install -e .
+          cd frauddetector/*outcome
+          curl -s -L https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip -o sam.zip
+          unzip -q sam.zip -d sam-install
+          sudo ./sam-install/install
+          sam local start-lambda &
+          pip install --quiet cloudformation-cli-java-plugin cloudformation-cli-python-plugin
```

Comment:
why do you need any of these? we can directly install dependencies like docker but to execute the resource we probably dont need plugin. Maybe `cloudformation-cli-python-lib` but even without it seems to be working

Reply:
Java resource provider couldn't build until `cloudformation-cli-java-plugin` was installed. Didn't test, but assumed Python resource provider might need more than `cloudformation-cli` installed as well 🤷‍♂️

Reply:
`rpdk.log` after `cfn submit --dry-run` fails for Python resource provider with only `cloudformation-cli` installed:
```shell
Traceback (most recent call last):
  File "/workspace/.pip-modules/lib/python3.8/site-packages/rpdk/core/cli.py", line 100, in main
    args.command(args)
  File "/workspace/.pip-modules/lib/python3.8/site-packages/rpdk/core/submit.py", line 14, in submit
    project.load()
  File "/workspace/.pip-modules/lib/python3.8/site-packages/rpdk/core/project.py", line 412, in load
    self.load_settings()
  File "/workspace/.pip-modules/lib/python3.8/site-packages/rpdk/core/project.py", line 201, in load_settings
    self.validate_and_load_resource_settings(raw_settings)
  File "/workspace/.pip-modules/lib/python3.8/site-packages/rpdk/core/project.py", line 230, in validate_and_load_resource_settings
    self._plugin = load_plugin(raw_settings["language"])
  File "/workspace/.pip-modules/lib/python3.8/site-packages/rpdk/core/plugin_registry.py", line 27, in load_plugin
    return PLUGIN_REGISTRY[language]()()
KeyError: 'python37'
```

Reply:
oh yea, that's right we need that for cfn generate etc